test/e2e/smoke/basic.yaml

```diff
@@ -3,7 +3,7 @@ kind: Workflow
 metadata:
   name: basic
   labels:
-    argo-e2e: true
+    argo-e2e: "true"
```

Comment:
this change indicates another change in behaviour - because we use the default unstructured converter - we must have valid metadata